test/fw/ptl/utils/pbs_logutils.py

```diff
@@ -369,6 +359,8 @@ def match_msg(self, lines, msg, allmatch=False, regexp=False,
                 dt_str = l.split(';', 1)[0]
                 if starttime is not None:
                     tm = self.convert_date_time(dt_str)
+                    self.logger.info(f'tm is {tm} | starttime is {starttime}'
```

Comment:
is this debugging message? if yes then please remove. If not then I suggest logging at debug level else this message will appear for each and every line passed to this method (i.e. `lines`) which will be too much info.

Reply:
Sorry, it failed in jenkins and I was trying to debug it here, will WIP PR

Reply:
It now passes in jenkins